CONTRIBUTING.rst

```diff
@@ -0,0 +1,151 @@
+============
```

Comment:
I think changes to README files should be done separately from such a refactor like this.

Reply:
Good point - this file should be added in separate PR. I think it would be really good to have such guide how to contribute to this project.